documentation/build.py

```diff
@@ -78,7 +78,7 @@ def __init__ ( self ):
         self.onLepka  = False
         self.onDocker = False
         hostname = socket.gethostname()
-        if hostname.startswith('lepka'): self.onLepka  = True
```

Comment:
We shouldn't have anything with lip6 specific hosts in the build code.

This is ok for now, as docs still need transitioning to meson

Reply:
Not even "LIP6" code. Build for my own laptop only. In order to copy the doc onto my local ``apache`` server so I can debug quick the doc (and have a local copy).

Reply:
I think we can do something a bit better than that ;) I'll look at a local server spin up for docs as part of the meson doc support.